src/libraries/System.Runtime.InteropServices.JavaScript.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project file should be deleted, or moved somewhere else. There are no project files under in src/libraries today and we want to keep it that way.

Reply:
This has been moved in recent commit.